Benjaminpjacobs/corkboard
# updates bootsrap gem

#### What does this PR do?
Fixes bootstrap/turbolink issues

#### Where should the reviewer start?
Gemfile, Application.js
#### How should this be manually tested?
Make sure the front page renders both logged in and logged out and that you can bid on a job withe modal

#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
  - Do Migrations Need to be ran?
no
  - Do Environment Variables need to be set?
no
  - Any other deploy steps?
bundle
